### DISCUSSION TITLE
token ttl

### DISCUSSION THREAD

**Discussion by yosiasz** [centreon-archive/centreon-archived#11220] [Q&A]
Greetings,

How long does the token last so that one needs to get a new token?

Thanks

**Comment by ponchoh** [comment]
For 21.04 I believe is 1 hour.

**Reply by yosiasz** [reply]
Is there a timestamp meta data that comes along for the ride?

**Reply by ponchoh** [reply]
```json
{"message":"Credentials have expired."}
```
